### DISCUSSION TITLE
Endre på ToggleGroup

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#1894] [Komponenter]
🔗 [Link til ToggleGroup i Aksel](https://aksel.nav.no/komponenter/core/toggle-group)

Vi ønsker å vurdere hva som kan forbedre brukeropplevelsen av ToggleGroup for utviklere og designere. Har du innspill til noe som vil gjøre denne komponenten bedre og mer brukervennlig for deg og brukerne av deres løsninger?

**Comment by Trent-NAV** [comment]
https://github.com/navikt/aksel/issues/1811

**Comment by sjur-gr** [comment]
Utkast til ny ToggleGroup @Trent-NAV 
<img width="935" alt="ToggleGroup demo" src="https://github.com/navikt/aksel/assets/82579704/fbe56b9a-3e16-42b5-9924-888661d802fc">

**Reply by Trent-NAV** [reply]
Takk for deling @sjur-gr Jeg syns det er en fin forbedring og adresserer mye av det som er nevnt i https://github.com/navikt/aksel/issues/1811  👍

Når det gjelder [**action fargene** ](https://nav-it.slack.com/archives/C7NE7A8UF/p1682071331133289) så kommer kanskje den endringen ved neste release? Da slipper vi problemet med at Deepblue både brukes som active state på CTA buttons og som selected state på toggle samt chips og pagination.

**Reply by sjur-gr** [reply]
Vi prøver å få til den endringen ja 👍
